test/test_face_areas.py

```diff
@@ -0,0 +1,24 @@
+import os
+import numpy as np
+import xarray as xr
+
+from unittest import TestCase
+from pathlib import Path
+
+import uxarray as ux
+
+current_path = Path(os.path.dirname(os.path.realpath(__file__)))
+
+class TestFaceAreas(TestCase):
```

Comment:
We write one test module (with possible several test classes with multiple test cases in each) per one code module (I just noticed though we oversaw this with `test_integrate.py` as well; we can fix that, too). 

So, could you please remove this test module, instead put this `TestFaceAreas ` class into the `test_grid.py` module?

Reply:
OK! I can do that for `test_integrate` as well in this PR. I guess it is the entire `class TestIntegrate` that needs to go in the other file?

Reply:
Ah, that's already done, I forgot to remerge main.

Reply:
That's correct; the entire class!